frontend/src/features/SideWindow/SideWindowLauncher.tsx

```diff
@@ -10,7 +9,6 @@ import { useAppSelector } from '../../hooks/useAppSelector'
 
 export function SideWindowLauncher() {
   const dispatch = useAppDispatch()
-  const newWindowRef = useRef() as MutableRefObject<HTMLDivElement>
   const { forceUpdate } = useForceUpdate()
 
   const missionState = useAppSelector(state => state.missionState)
```

Comment:
Suite à ton commentaire @claire2212 je regarde les sélecteurs et je me dis qu'on peut encore limiter le nombre de rendus.
```suggestion
  const missionStateIsFormDirty = useAppSelector(state => state.missionState.isFormDirty)
```

idem pour `sideWindow`. Et `hasAtLeastOneMissionFormDirty`pourrait être directement mis dans useAppSelector, voire créer un selecteur dédié en dehors du composant.

A voir pour les refacto de janvier ?